cli/src/config.rs

```diff
@@ -949,6 +954,8 @@ pub struct GenesisEntry {
     pub address: String,
     // Filepath to the compiled program to embed into the genesis.
     pub program: String,
+    // Whether the genesis program is upgradeable.
+    pub upgradeable: bool,
```

Comment:
This should be an optional field otherwise it breaks the existing `Anchor.toml` files with genesis entries. https://github.com/coral-xyz/anchor/actions/runs/6386297008/job/17343757677?pr=2642#step:10:297

```
Error: TOML parse error at line 1, column 1
  |
1 | [[test.genesis]]
  | ^^^^^^^^^^^^^^^^
missing field `upgradeable`
```

Reply:
ahh my bad again, fixed!